lib/hutch/tracers.rb

```diff
@@ -0,0 +1,14 @@
+module Hutch
+  module Tracers
+    autoload :NullTracer, 'hutch/tracers/null_tracer'
+    autoload :NewRelic,   'hutch/tracers/newrelic'
+
+    TRACERS = {
```

Comment:
So how does one extend this hash? I recommend doing without all this Rails-y DSL stuff and just providing a class.